src/styles/components/card.scss

```diff
@@ -0,0 +1,101 @@
+@import "./utilits/colors.scss";
+
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white-color;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-left: 19px;
+    margin-right: 19px;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    height: 100%;
+    img {
+      width: 160px;
+      height: 134px;
+      display: block;
+      width: 100%;
+      object-fit: cover;
+    }
```

Comment:
Don't use tags for styling, use classes instead.
Tags are used only to reset or normalize styles.